submissions/rubyAdeniregun/todo-app/src/components/style.css

```diff
@@ -0,0 +1,47 @@
+.todoItem {
```

Comment:
Why do you have 3 css files? I think one file is enough. Add the single css file to index.js and your styles will be fine.